client/my-sites/plans/main.jsx

```diff
@@ -36,9 +41,54 @@ const Plans = React.createClass( {
 		};
 	},
 
+	shouldDisplayJetpackPlansDisabled() {
+		const { selectedSite } = this.props;
+
+		return selectedSite && selectedSite.jetpack && ! isEnabled( 'manage/jetpack-plans' );
+	},
+
+	componentDidMount() {
+		const { context, selectedSite } = this.props,
+			analyticsPageTitle = 'Plans',
+			basePath = route.sectionify( context.path ),
+			analyticsBasePath = ( selectedSite ) ? basePath + '/:site' : basePath;
+
+		// FIXME: Auto-converted from the Flux setTitle action. Please use <DocumentHead> instead.
+		context.store.dispatch( setTitle( i18n.translate( 'Plans', { textOnly: true } ) ) );
+
+		if ( this.shouldDisplayJetpackPlansDisabled() ) {
+			analytics.pageView.record( basePath + '/jetpack/:site', analyticsPageTitle + ' > Jetpack Plans Not Available' );
```

Comment:
If you're comfortable with adding the change to the diff, we could use the `<PageViewTracker />` component or [recordPageView](https://github.com/Automattic/wp-calypso/blob/master/client/state/analytics/actions.js#L61) redux actions.